vm/vm.go

```diff
@@ -20,11 +21,23 @@ func Run(program *Program, env any) (any, error) {
 	if program == nil {
 		return nil, fmt.Errorf("program is nil")
 	}
-
 	vm := VM{}
 	return vm.Run(program, env)
 }
 
+func RunWithConfig(program *Program, env any, config *conf.Config) (any, error) {
```

Comment:
Lets not expose additional API. Actually `*vm.VM` can be used directly! I will update the code.

Reply:
Makes sense! A separate PR then?

Reply:
I already updated the code)

Reply:
Sorry - maybe an issue with Github mobile app 😅